### PR TITLE
SRM release notes for release 788

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/service-reliability-management.md [hunk 1]
```diff
@@ -14,6 +14,46 @@ Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS r
 :::
 
 
+## March 23, 2023, version 788xx
+
+### What's new
+
+- When you try editing an existing SLO, you will see the **Tags (optional)** field autopopulated even if no value was entered while creating the SLO. Harness uses the selected SLI type as value to autopopulate the **Tags (optional)** field. (SRM-14121)
+  
+- A new user interface (UI) for Elasticsearch health source configuration. This update is designed to improve the overall user experience and make it easier to configure Elasticsearch health sources. (SRM-14180)
```
Comment (StrawberryPad) [inline release-notes/service-reliability-management.md:23]: **There is a** new user interface (UI)...

File: release-notes/whats-new.md [hunk 2]
```diff
@@ -16,6 +16,14 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+
+## March 15, 2023, version 788xx
+
+### Service Reliability Management
+- When you try editing an existing SLO, you will see the **Tags (optional)** field autopopulated even if no value was entered while creating the SLO. Harness uses the selected SLI type as value to autopopulate the **Tags (optional)** field. (SRM-14121)
+  
+- A new user interface (UI) for Elasticsearch health source configuration. This update is designed to improve the overall user experience and make it easier to configure Elasticsearch health sources. (SRM-14180)
```
Comment (StrawberryPad) [inline release-notes/whats-new.md:25]: Same as above comment

**Review by StrawberryPad — DISMISSED** [review]
Just one suggestions (x2)

**Review by StrawberryPad — DISMISSED** [review]

**Review by charanya-j — DISMISSED** [review]

**Comment by SudheendraKatte** [comment]
Resolved merge conflict.

**Pull request merged**